idea_inspection.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/sh -e
+
+#################################################
+# IntelliJ IDEA inspections for checkstyle.
+#################################################
```

Comment:
please add example of usage by single line command.

My try:
```
$ export IDEA_HOME="/home/rivanov/java/idea-IU-171.4249.39/bin/idea.sh" 

$ echo $IDEA_HOME
/home/rivanov/java/idea-IU-171.4249.39/bin/idea.sh

$ ./idea_inspection.sh
./idea_inspection.sh: 16: ./idea_inspection.sh: [[: not found
IDEA_HOME variable not found.
```

Reply:
done